### ISSUE TITLE
[BUG] Set-PnPWebTheme returns "Unknwon Error" since yesterday

### ISSUE THREAD

**Issue by zapftho** [pnp/powershell#4528]
### Reporting an Issue or Missing Feature
Since about yesterday evening we are getting errors when using Set-PnPWebTheme command in our Teams Provisioning. The issue started without any changes from our end and I cannot find a reason for it so far. Is anyone facing the same issue? I have also checked if the Theme is still there using Get-PnPTenantTheme and can confirm that this is the case.

**EDIT: After additional tests I found out that the same error occurs when using the "Microsoft.Online.SharePoint.PowerShell" module from Microsoft. So this is probably not related to PnP Powershell. I will create a Ticket at Microsoft as well just to be sure.**

### Expected behavior 
Set-PnPWebTheme completes without error and updates the Web Theme.

### Actual behavior 

![image](https://github.com/user-attachments/assets/f24a3d55-1a8c-414f-9ddf-4f052cedb970)

Result of Get-PnPException:

![image](https://github.com/user-attachments/assets/18482792-d5ad-48ce-8824-c9b690318638)

### Steps to reproduce behavior

Connect-PnPOnline -Url "<your sharepoint site Url>" -ClientId "<your client id>" -Interactive
Set-PnPWebTheme -Theme "<your theme name>"

### What is the version of the Cmdlet module you are running?
2.12.0

### Which operating system/environment are you running PnP PowerShell on?
- [ ] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [x] Azure Functions
- [ ] Other : please specify

**Comment by veronicageek** [comment]
Closing for now. When MSFT fixes it, it will work for PnP as well.

**Comment by zapftho** [comment]
The Microsoft issue is still not fixed but I have been able to find a workaround. So far it turned out that the issue caused by using RGBA values in the color palette. We can also confirm that this started at some point in both of our Tenants. Not at the same time but with some days difference. It even started earlier in our Standard Release tenant than in our Targeted Release Tenant...

This will no longer work and cause the error described in this issue:

```
$themepalette = @{
   ...
    "whiteTranslucent40" = "rgba(173, 173, 173, 0.7)";
   ...
}
```

As this is officially used also in Microsoft Docs I assume this is a bug by them. However, I found a workaround which does solve it for us. INstead of using the RGBA value we have now set an 8-digits HEX value which does contain opacity as well. I got this idea after finding this in the documentation about the classic theming: https://learn.microsoft.com/en-us/sharepoint/dev/general-development/color-palettes-and-fonts-in-sharepoint

![image](https://github.com/user-attachments/assets/9f2ef43c-e88e-4774-9f64-35a61e279199)

So this is what we replaced it with in our case to get it back working again:

```
$themepalette = @{
   ...
    "whiteTranslucent40" = "#b3adadad";
   ...
}
```
